drivers/ovsdriver.go

```diff
@@ -204,6 +209,23 @@ func (d *OvsDriver) DeleteNetwork(id, encap string, pktTag, extPktTag int, gatew
 		sw = d.switchDb["vlan"]
 	}
 
+	// Delete infra nw endpoint if present
+	if nwType == "infra" {
+		hostName, _ := os.Hostname()
```

Comment:
for my understanding? Why is this required? Wouldn't we receive endpoint delete from need?